src/ble/CHIPBleServiceData.h

```diff
@@ -70,7 +70,7 @@ struct ChipBLEDeviceIdentificationInfo
     uint16_t GetDeviceDiscriminator(void)
     {
         uint16_t discriminator                = chip::Encoding::LittleEndian::Get16(DeviceDiscriminator);
```

Comment:
CHIP seems to be very poor when it comes to coding standard at the moment, but looks like "m" prefix is used more in case of private members (though it's not a general rule).